master/buildbot/test/unit/test_schedulers_forcesched.py

```diff
@@ -390,7 +390,7 @@ def do_ParameterTest(self,
                     import xerox
                     formated = self.formatJsonForTest(gotJson)
                     print(
-                        "You may update the test with (copied to clipboard):\n" + formated)
+                        "You may update the test with (copied to clipboard):\n" + formatted)
```

Comment:
please see above the actual variable name

Reply:
@sa2ajj nice catch. sorry to miss it. Any possibility you'd consider changing the variable name to the correct spelling ?

Reply:
@luzpaz of course, please fix the variable name

Reply:
done. re-pushed with requested revision.